helm/idol-nifi/templates/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,58 @@
+# BEGIN COPYRIGHT NOTICE
```

Comment:
Should we make all the `prometheus` bits conditional on `nifi.autoScaling.enabled` (and or some other separate control if it's useful to have the reporting for other purposes but you don't want to run autoscale?)

Reply:
Added two new enabled flags:
    nifiMetrics.enabled - enables/disables nifi custom metrics
    nifiPrometheus.enabled - enables/disables nifi prometheus monitoring